### ISSUE TITLE
Cannot configure "auth-password-policy" when "storage-fs" is enabled

### ISSUE THREAD

**Issue by kevinmcox** [osTicket/osTicket-plugins#225]
- osTicket 1.16.1 (clean install)
- Attachments on the filesystem 0.3
- Password Management Policies 0.1

If both the "Attachments on the filesystem" plugin and the "Password Management Policies" plugins are enabled, I am unable to configure the latter.

Attempting to do so just displays the configuration options for the filesystem plugin. The UI does open the different URL for each (plugins.php?id=1 vs. plugins.php?id=2) and the correct name is shown, just not the options. See attachment.

Disabling the filesystem plugin does reveal the correct configuration options for password policies.

![Plugins](https://user-images.githubusercontent.com/25358601/154608836-4b8d0906-f3ab-49c3-b508-c781433d965b.png)

**Comment by JediKev** [comment]
@kevinmcox 

You are using PHP 8.1 and none of the current versions of osTicket support that. v1.16.1 requires PHP 8.0. You must downgrade your PHP and you will be good.

Cheers.

**Comment by kevinmcox** [comment]
Thanks for the fast reply @JediKev, that was exactly it. When I upgraded to 1.16 and read `PHP version 8.0 (or greater)` I went all the way up to 8.1.

I do see now that the ReadMe was [later edited](https://github.com/osTicket/osTicket/commit/fdc044519c4412b1fd12cad1e1e15203c8c04c6b) to specify 8.0, but I missed that change in the 1.16.1 release notes.

I downgraded PHP and it works as expected, thanks again.
